backend/src/main/java/gov/cdc/usds/simplereport/api/ExceptionWrappingManager.java

```diff
@@ -18,6 +18,10 @@
 
     @ExceptionHandler(AccessDeniedException.class)
     public GraphQLError wrapSecurityExceptions(AccessDeniedException e) {
+        // Add This to `application-local.yaml` to debug what's gone wrong.
```

Comment:
Do we have a troubleshooting wiki or section on the README? This would probably be more useful there. Putting it in application-local.yaml is my first reflex and that's probably a totally useless, since nobody looks there unless they already know the answer.

Reply:
I don't think `application-local.yaml` is in git. I figured that when encountering the error, someone might search the code for the exception string. For commenting, is there somewhere else we document what logging is good to enable as a dev? Maybe adding it to a commented out section of `application-dev.yaml`?

Reply:
Yes, quite right--that's the whole point of having it be a separate profile. There is a commented-out section in application-dev that is probably what I was thinking of, which is slightly better but still not awesome. 

Here's a silly idea: put it in a DEBUG statement here?

Reply:
Oh, no, I meant putting the suggestion to turn on spring security debugging into our debug log. 😺